packages/unified-latex-util-arguments/tests/attach-arguments-in-array.test.ts

```diff
@@ -197,5 +197,120 @@ describe("unified-latex-util-arguments", () => {
                 ],
             },
         ]);
+
+        // multiple delimiters
+        nodes = strToNodes("\\xxx_a^_^\\xxx__");
+        attachMacroArgsInArray(nodes, { xxx: { signature: "r__" } });
+        expect(nodes).toEqual([
+            {
+                type: "macro",
+                content: "xxx",
+                args: [
+                    {
+                        type: "argument",
+                        content: [{ type: "string", content: "a^" }],
+                        openMark: "_",
+                        closeMark: "_",
+                    }
+                ],
+            },
+            { type: "string", content: "^" },
+            {
+                type: "macro",
+                content: "xxx",
+                args: [
+                    {
+                        type: "argument",
+                        content: [],
+                        openMark: "_",
+                        closeMark: "_",
+                    }
+                ],
+            }
+        ]);
+
+        // embellishments
+        nodes = strToNodes("\\xxx^a\\xxx_b\\xxx_b^a");
```

Comment:
Can you add another test with `\xxx^{a_b}`?

Reply:
Done in https://github.com/siefkenj/unified-latex/pull/42/commits/5839b7364cbe623781d11f501a7bf1ccfd3c4db1